.github/workflows/validateGithubActions.yml

```diff
@@ -25,4 +25,8 @@ jobs:
 
             # Rebuild all the actions on this branch and check for a diff. Fail if there is one,
             # because that would be a sign that the PR author did not rebuild the Github Actions
-            - run: ./.github/scripts/verifyActions.sh
+            - name: Verify Javascript Action Builds
+              run: ./.github/scripts/verifyActions.sh
+
+            - name: Validate actions and workflows
+              run: npm run gh-actions-validate
```

Comment:
I'm sure it's for consistency with the rest but why did we decide to proxy all our shell scripts with NPM actions ?

Reply:
It's so you can run them from any directory in the project and they will still work the same. Like it I were to do `cd .github && ./scripts/validateActionsAndWorkflows.sh`, then the script would fail because it tries to run `cd ./.github` and that would fail. But if you run it as an npm script then it's always running from the project root.

Reply:
Hmm, there are other ways to ensure that in Bash without requiring `npm`. In this case, this we already use it that's probably fine but we shouldn't rely on it otherwise IMO.